### ISSUE TITLE
sbJSON itemLink associationType not translated

### ISSUE THREAD

**Issue by jlblcc** [adiwg/mdTranslator#137]
It appears the itemLink associationTypes are not making it through the translation.

Example SB item:https://www.sciencebase.gov/catalog/item/5542394be4b0a658d793b55a

**Comment by stansmith907** [comment]
Yep, I see that I postponed this section because it was (is) a PINA and forgot to place a TODO tag.  I just skipped it.  Anyway, in there now.  

Here are few item to contemplate...
 * remember these do not go back into sbJson through the sbJson writer.  The writer ignores building relatedItems, thus associated resources.  
 * There are two sbJson linkTypes we cannot translate; copyFrom/copyInto, precededBy/succeededBy.  I just moved these values into the translator as is.  However these were not added to the codelist.
 * All associations entered into the translator are expressed as **forward** relationships.  To accomplish this any sbJson reverse relationships are reversed by the reader.
 * If sbJson *link.type* is missing or empty the association type is set to 'missing' since the associationType is required.
